com.ozmartians.VidCutter.json

```diff
@@ -0,0 +1,302 @@
+                    "cd source/build-8 && make",
+                    "cd source/build-8 && make install"
+                ],
+                "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://bitbucket.org/multicoreware/x265/downloads/x265_2.7.tar.gz",
+                    "sha256": "d5e75fa62ffe6ed49e691f8eb8ab8c1634ffcc0725dd553c6fdb4d5443b494a2"
+                }]
+            },
+            {
+                "name": "fdkaac",
+                "config-opts": [ "--disable-static" ],
+                "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/mstorsjo/fdk-aac/archive/v0.1.6.tar.gz",
+                    "sha256": "adbcd793e406e1b88b3c1c41382d49f8c27371485b823c0fdab69c9124fd2ce3"
+                }]
+            },
+            {
+                "name": "lame",
+                "config-opts": [ "--enable-shared", "--disable-static", "--enable-nasm" ],
+                "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/lame/lame-3.100.tar.gz",
+                    "sha256": "ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e"
+                }]
+            }]
+        }]
+    },
+    {
+        "name": "mediainfo",
+        "buildsystem": "simple",
+        "cleanup": ["/include", "/lib/pkgconfig", "/share"],
+        "build-commands": [
+            "mv lib/libmediainfo.so* lib/libzen.so* ${FLATPAK_DEST}/lib -v",
+            "install -D bin/mediainfo ${FLATPAK_DEST}/bin/mediainfo"
+        ],
+        "sources": [
+        {
+            "type": "archive",
+            "url": "https://mirror.pseudoform.org/community/os/x86_64/libzen-0.4.37-1-x86_64.pkg.tar.xz",
+            "sha256": "2ed43babf6aea81fce788426a57ddd8769ab59129446b2058460b5632b724e04"
+        },
+        {
+            "type": "archive",
+            "url": "https://mirror.pkgbuild.com/community/os/x86_64/libmediainfo-18.03.1-1-x86_64.pkg.tar.xz",
+            "sha256": "c80fd36769e65a607d5ea6d422e5de01946ed53d4b48db05f992291cc277e9a4"
+        },
+        {
+            "type": "archive",
+            "url": "https://mirror.pkgbuild.com/community/os/x86_64/mediainfo-18.03.1-1-x86_64.pkg.tar.xz",
+            "sha256": "066be192f1123bff9d33fb7beaee394d8daa0d0345a81ded29f306587f2af119"
+        }],
+        "modules": [
+        {
+            "name": "libmms",
+            "config-opts": ["--disable-static"],
+            "sources": [
+            {
+                "type": "archive",
+                "url": "https://download.sourceforge.net/libmms/libmms-0.6.4.tar.gz",
+                "sha256": "3c05e05aebcbfcc044d9e8c2d4646cd8359be39a3f0ba8ce4e72a9094bee704f"
+            }]
+        }]
+    },
+    {
+        "name": "vidcutter",
```

Comment:
You have a mime file but you need to rename it to match your app-id (and the icon it references) to be exported.

Reply:
@TingPing this is done with latest update... just an FYI, no additional icon is required for the mime-type as it refers to the app name (which has had the icon renamed to app-id). App icon from 32x32 and below is a different version to the larger app icon/tile, suited for file managers and the like. So com.ozmartians.VidCutter.png is in various folders under hicolor theme with the desktop + mime file both referencing the same icon name.

And the mime-type file for .wtv has been merged into the com.ozmartians.VidCutter.xml mime file.

Cheers!